hexa/workspaces/views.py

```diff
@@ -35,6 +35,13 @@ def credentials(request: HttpRequest, workspace_slug: str) -> HttpResponse:
     env = {}
     for connection in connections:
         env.update(connection.env_variables)
+        if connection.connection_type == ConnectionType.POSTGRESQL:
```

Comment:
It's weird to have the special case for this connection type here. It should be in the property "env_variables" of the connection.